types/v56/openapi.go

```diff
@@ -155,7 +155,8 @@ type NetworkProvider struct {
 	ID   string `json:"id"`
 }
 
-// VdcComputePolicy is represented as VM sizing policy in UI
+// VdcComputePolicy contains VDC specific configuration for workloads. (version 1.0.0)
+// Use VdcComputePolicyV2 instead (version 2.0.0)
```

Comment:
I think it is still valid. We don't want someone to use this type directly because we maybe just remove it in the future and shift all these fields to `VdcComputePolicyV2`

Reply:
Changed.